x-pack/plugins/enterprise_search/server/routes/app_search/log_settings.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { IRouteDependencies } from '../../plugin';
+
+const logSettingsSchema = schema.object({
```

Comment:
Do we anticipate multiple POST/PUT endpoints using this schema in the future? In the credentials router file, I pulled a var out to the top primarily because it was super complex & reused between 2 different endpoints - in this case we could just leave this explicitly on line 44.

But also no objections/no strong feelings if we like leaving it at the top of files so it's the first thing people see 🤷

Reply:
bf8acd5aad2c90b18b3dd50252413ac5c1d937e6